lib/domain/enums/RunDetectorQualities.js

```diff
@@ -15,6 +15,7 @@
 const RunDetectorQualities = {
     GOOD: 'good',
     BAD: 'bad',
+    NONE: 'none',
```

Comment:
This enum must reflect what is in the database, and if you add this value it does not. I think you should instead handle the case of null value directly.